### PR TITLE
Fixes for key definitions and examples

### LINKED ISSUE #12: Confusion between CEK and JWE Encryption Key
The spec at https://w3c.github.io/webpayments-crypto/ describes the CEK as an RSA public key but this is in fact the JWE Encryption Key (which encrypts the symmetric CEK) for exchange.

### PR DESCRIPTION
**Pull request by adrianhopebailie**
Fixes #12

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,18 +288,18 @@ <h2>
       </h2>
       <section id="key">
         <h3>
-          Content Encryption key
+          Key Encryption key
```
Comment (wanli) [inline index.html:291]: I am a little confused -- content encryption key is referred in a few places in the doc: line 224, 370, 460. Why do we need a "Key Encryption Key" section here?

**Comment by ianbjacobs** [comment]
@adrianhopebailie, just a note: I really appreciate the edits. I am merging though I have some questions. I think it will be easier to work on the fully-merged documents. (And I plan to fix minor editorial things.)

File: index.html (hunk 1 above)
Comment (adrianhopebailie) [reply index.html:291]: @wanli There are two keys. The RSA key is not the CEK it is just used to securely exchange the CEK with the other party. The CEK is a symmetric key used to encrypt the content using AES 128 GCM.

Comment (wanli) [reply index.html:291]: I see. So the " Encrypted Key" in the message is from the encrypted CEK. Thanks the for explantation!

**Pull request merged**